semtypes/src/main/java/io/ballerina/types/typeops/ListOps.java

```diff
@@ -386,12 +379,11 @@ public static SemType bddListMemberType(Context cx, Bdd b, SubtypeData key, SemT
         } else {
             BddNode bddNode = (BddNode) b;
             return Core.union(bddListMemberType(cx,
```

Comment:
This is easy on the eyes for me. wdyt?
```suggestion
            return Core.union(bddListMemberType(cx, bddNode.left(), key,
```

Reply:
Fixed